src/lib.rs

```diff
@@ -121,4 +121,246 @@ impl zed::Extension for ScalaExtension {
     }
 }
 
+use zed_extension_api::{DebugAdapterBinary, DebugTaskDefinition, StartDebuggingRequestArgumentsRequest, Worktree};
+
+impl ScalaExtension {
+    fn get_metals_dap_path(&self, worktree: &zed::Worktree) -> Result<String> {
+        worktree
+            .which("metals-dap")
+            .ok_or_else(|| "Metals DAP server (metals-dap) must be installed manually. Please ensure it is in your PATH.".to_string())
```

Comment:
Hm, is `metals-dap` a thing? I just never heard of it.

It also doesn't exist as a project in metals SBT build, and https://github.com/scalacenter/scala-debug-adapter doesn't publish it either.